docs/website/docs/plus/getting-started/tutorial.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Projects tutorial
+description: Using the dlt+ cli commands to create and manage dlt+ Projects
+keywords: [command line interface, cli, dlt init, dlt+, projects]
+---
+
+This tutorial introduces you to dlt+ Projects and the essential cli commands needed to create and manage them. You will learn how to:
+
+* initialize a new dlt+ Project
+* navigate the `dlt.yml` file
+* add sources, destinations, and pipelines
+* run pipelines using cli commands
+* inspect datasets
+* work with dlt+ Profiles for enabling different configurations
+
+## Prerequisites
+
+- Python 3.9 or higher installed
+- Virtual environment with installed dlt+ package ([installation guide](./installation.md))
+- Valid dlt+ license (EULA)
+- Make sure you're familiar with the [core concepts of dlt](../../reference/explainers/how-dlt-works.md)
+
+:::tip
+You can find the full list of available cli commands under [cli reference](../reference.md)
+:::
+
+## Creating a new dlt+ Project
+
+### Generating your first project
+Start by creating a new folder for your project. Then, navigate to the folder in your terminal.
+
+```sh
+mkdir tutorial && cd tutorial
+```
+
+Run the following command to initialize a new dlt+ Project:
+
+```sh
+# Initialize a dlt+ Project named "tutorial", the name is derived from the folder name
+dlt project init arrow duckdb
+```
+
+This command generates a project named `tutorial` with:
+- one [pipeline](../../general-usage/pipeline)
+- one Arrow source defined in `sources/arrow.py`
+- one DuckDB destination
+- one dataset on the DuckDB destination
+
+### The generated folder structure
+After running the command, the following folder structure is created:
+
+```sh
+.
+├── .dlt/                 # your dlt settings including profile settings
+│   ├── config.toml
+│   ├── dev.secrets.toml
+│   └── secrets.toml
+├── _storage/             # local storage for your project, excluded from git
+├── destinations/         # your destinations, empty in this example
```

Comment:
I thought we don't generate the destinations folder anymore. And also I think a `requirements.txt` gets added too